src/components/cart/cart-item/cart-item.module.scss

```diff
@@ -8,13 +8,13 @@
 
 .itemContent {
     display: grid;
-    grid-template-columns: 75px 1fr;
+    grid-template-columns: 100px 1fr;
```

Comment:
Small fix. I didn't notice that image size in cart item are different in their "narrow" and "wide" versions.